src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1259,7 +1259,7 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMProposeSet> const& m)
             return;
         }
 
-        if (app_.getFeeTrack ().isLoadedLocal ())
+        if (app_.getFeeTrack ().isLoadedLocal () && ! cluster())
```

Comment:
μ-nit: prefer `if (! cluster && app_.getFeeTrack().isLoadedLocal())` since we can avoid grabbing a lock in `getFeeTrack` when dealing with a cluster member.